### PR TITLE
fidals/shopelectro#392 static files forwarding

### PR DESCRIPTION
**Pull request by artemiy312**
Closes fidals/shopelectro#392

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -87,7 +87,7 @@ pipeline:
       - docker-compose -f docker-compose-build.yml build --no-cache nodejs python-dev python-prod
     volumes:
       - /var/run/docker.sock:/var/run/docker.sock
-      - /tmp/cache/drone/stroyprombeton/front_build/${DRONE_COMMIT_SHA}:/usr/app/front/build
+      - /tmp/cache/drone/stroyprombeton/front_build/${DRONE_COMMIT_SHA}:/drone/src/github.com/fidals/stroyprombeton/front/build
```
Comment (duker33) [inline .drone.yml]: it's not clear why we are needed this mapping.
`/drone/src/github.com/fidals/stroyprombeton/front/build` should be visible from every pipeline container according to doc http://docs.drone.io/workspace/

File: docker/docker-compose.yml [hunk 2]
```diff
@@ -22,6 +22,8 @@ services:
     networks:
       - stb-backend
       - stb-frontend
+    # @todo #236:60m Remove app service's depenency from a nodejs servise.
```
Comment (duker33) [inline docker/docker-compose.yml]: `depenency` -> `dependency`
`servise` -> `service`.

Add plz `se2` at the end of title

`30 mins` will be enough for this task, i think. It's only about volumes

File: docker/docker-compose.yml [hunk 3]
```diff
@@ -36,6 +38,7 @@ services:
 
   nodejs:
     image: fidals/stb-nodejs:dev
+    command: gulp watch
```
Comment (duker33) [inline docker/docker-compose.yml]: it's not clear if we should do build or watch by default.
So, the best way is to rm this command and add related recipes to Makefile.
We'll enforce explicitness in this case

**Comment by duker33** [comment]
@artemiy312 , i mention that you should create symmetric PR for SE side. This stb commit closes the task from SE

File: .drone.yml (hunk 1 above)
Comment (artemiy312) [reply .drone.yml]: @duker33 We alreade have an [issue](https://github.com/fidals/refarm-site/issues/132) to resolve it

Comment (duker33) [reply .drone.yml]: @artemiy312 , but you reused this code here :)

Ok, let's discuss it in task https://github.com/fidals/refarm-site/issues/132

Comment (artemiy312) [reply .drone.yml]: @duker33 
>The base attribute defines a shared base volume available to all pipeline steps. This ensures your source code, dependencies and compiled binaries are persisted and shared between steps.
 
I do not know default value of `base` attribute

**Pull request merged**